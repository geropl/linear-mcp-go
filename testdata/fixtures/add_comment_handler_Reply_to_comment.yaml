---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 322
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tquery GetIssueByIdentifier($teamKey: String!, $number: Float!) {\n\t\t\tissues(filter: { team: { key: { eq: $teamKey } }, number: { eq: $number } }, first: 1) {\n\t\t\t\tnodes {\n\t\t\t\t\tid\n\t\t\t\t\tidentifier\n\t\t\t\t\ttitle\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"number":10,"teamKey":"TEST"}}'
        form: {}
        headers:
            Content-Type:
                - application/json
            User-Agent:
                - linear-mcp-go/1.0.0
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"data":{"issues":{"nodes":[{"id":"1c2de93f-4321-4015-bfde-ee893ef7976f","identifier":"TEST-10","title":"Updated Test Issue"}]}}}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"82-w0K/VnjlqJtYAurPyBwU/9QgAFo"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 0s
    - id: 1
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 575
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tmutation AddComment($input: CommentCreateInput!) {\n\t\t\tcommentCreate(input: $input) {\n\t\t\t\tsuccess\n\t\t\t\tcomment {\n\t\t\t\t\tid\n\t\t\t\t\tbody\n\t\t\t\t\turl\n\t\t\t\t\tcreatedAt\n\t\t\t\t\tuser {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tname\n\t\t\t\t\t}\n\t\t\t\t\tissue {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tidentifier\n\t\t\t\t\t\ttitle\n\t\t\t\t\t\turl\n\t\t\t\t\t}\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"input":{"body":"This is a reply to the comment","issueId":"1c2de93f-4321-4015-bfde-ee893ef7976f","parentId":"ae3d62d6-3f40-4990-867b-5c97dd265a40"}}}'
        form: {}
        headers:
            Content-Type:
                - application/json
            User-Agent:
                - linear-mcp-go/1.0.0
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"data":{"commentCreate":{"success":true,"comment":{"id":"272b238c-8065-4b61-975c-903b2fb9825a","body":"This is a reply to the comment","url":"https://linear.app/linear-mcp-go-test/issue/TEST-10/updated-test-issue#comment-272b238c","createdAt":"2025-03-30T14:16:58.457Z","user":{"id":"cc24eee4-9edc-4bfe-b91b-fedde125ba85","name":"Gero Leinemann"},"issue":{"id":"1c2de93f-4321-4015-bfde-ee893ef7976f","identifier":"TEST-10","title":"Updated Test Issue","url":"https://linear.app/linear-mcp-go-test/issue/TEST-10/updated-test-issue"}}}}}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"219-zDLAT8aClvacOYBCVUyaeiMboLM"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 0s
