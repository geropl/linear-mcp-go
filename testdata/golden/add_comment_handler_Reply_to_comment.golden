err: ""
output: |-
    Added comment to Issue: TEST-10 (UUID: 1c2de93f-4321-4015-bfde-ee893ef7976f)
    Thread: ae3d62d6-3f40-4990-867b-5c97dd265a40
    Comment: 272b238c-8065-4b61-975c-903b2fb9825a
    URL: https://linear.app/linear-mcp-go-test/issue/TEST-10/updated-test-issue#comment-272b238c
